examples/Vendored-XCFramework12-Example/VendoredXCFrameworkExample.xcodeproj/project.pbxproj

```diff
@@ -231,7 +231,7 @@
+			shellScript = "archivePath=\"${BUILD_DIR}/VendoredXCFrameworkExample.xcarchive\"\n\nxcodebuild archive \\\n   -workspace \"${PROJECT_DIR}/Examples.xcworkspace\" \\\n   -scheme \"VendoredXCFrameworkExample\" \\\n   -configuration \"Release\" \\\n   -archivePath \"${archivePath}\" \\\n   CODE_SIGN_IDENTITY=\"\" \\\n   CODE_SIGNING_REQUIRED=NO \\\n   CODE_SIGN_ENTITLEMENTS=\"\" \\\n   CODE_SIGNING_ALLOWED=\"NO\"\n\n# ---------------------------------------\n# Collecting debug symbols in archive\n# ---------------------------------------\nexitCode=0\n\nbcSymbolMapsPath=\"${archivePath}/BCSymbolMaps\"\n\n# Ensure the expected Xcode 11-style BananaLib BCSymbolMaps are in the root of the archive\nlibraryBCSymbolMapDir=\"${PROJECT_DIR}/BananaLib/build/BananaLib.xcframework/ios-arm64/BananaLib.framework/BCSymbolMaps\"\nfor file in \"${libraryBCSymbolMapDir}\"/*; do\n    filename=\"${file##*/}\"\n    if ! test -f \"${bcSymbolMapsPath}/${filename}\"; then\n        echo \"error: Missing BCSymbolMap for BananaLib: ${filename}\"\n        exitCode=1\n    fi\ndone\n\n# Ensure the expected Xcode 12-style CoconutLib BCSymbolMaps are in the root of the archive\nlibraryBCSymbolMapDir=\"${PROJECT_DIR}/CoconutLib/build/CoconutLib.xcframework/ios-arm64/BCSymbolMaps\"\nfor file in \"${libraryBCSymbolMapDir}\"/*; do\n    filename=\"${file##*/}\"\n    if ! test -f \"${bcSymbolMapsPath}/${filename}\"; then\n        echo \"error: Missing BCSymbolMap for CoconutLib: ${filename}\"\n        exitCode=1\n    fi\ndone\n\n# Ensure the expected dSYMs are in the root of the archive\ndsymsPath=\"${archivePath}/dSYMs\"\n\nif ! test -d \"$dsymsPath/VendoredXCFrameworkExample.app.dSYM\"; then\n    echo \"error: Missing dSYM: VendoredXCFrameworkExample.app.dSYM\"\n    exitCode=1\nfi\n\nif ! test -d \"$dsymsPath/CoconutLib.framework.dSYM\"; then\n    echo \"error: Missing dSYM: CoconutLib.framework.dSYM\"\n    exitCode=1\nfi\n\n# ------------------------------------------------------\n# Stripping embedded Xcode 11-style BananaLib framework\n# ------------------------------------------------------\n\n# Ensure BCSymbolMaps have been stripped from the embedded Xcode 11-style BananaLib framework\n\nappPath=\"${archivePath}/Products/Applications/VendoredXCFrameworkExample.app\"\nframeworkPath=\"${appPath}/Frameworks/BananaLib.framework\"\n\nif test -d \"${xcode11FrameworkPath}/BCSymbolMaps\"; then\n    echo \"error: Embedded BCSymbolMap not removed: BananaLib.framework/BCSymbolMaps\"\n    exitCode=1\nfi\n\nexit $exitCode\n";
```

Comment:
can you pull this out into a separate file? it's difficult to review as a single line

Reply:
Good call, yep will do.